app/controllers/questions/asset_sale_gate_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Questions
+  class AssetSaleGateController < TicketedQuestionsController
```

Comment:
Might be more consistent to just name this `AssetSaleController` or `SoldAssetsController` to be consistent with the rest of the controllers. We don't have any other explicit `Gate` controllers.

Reply:
would love thoughts on naming convention as there will be more gate controllers, but still not sure if it'll be clearer to folks if they are explicitly named "*Gate" or if I try to come up with a descriptive name for each. Designs [here]([https://www.figma.com/file/KhjUvgH13LaUjOHR8sTuRm/Get-Your-Refund?node-id=3503%3A0])

Reply:
went ahead and did this, refactored all the things (controller + form) from AssetSalesGate to SoldAssets